chemprop/features/featurization.py

```diff
@@ -350,17 +365,22 @@ def __init__(self, mol: Union[str, Chem.Mol, Tuple[Chem.Mol, Chem.Mol]],
             ri2pi, pio, rio = map_reac_to_prod(mol_reac, mol_prod)
            
             # Get atom features
-            f_atoms_reac = [atom_features(atom) for atom in mol_reac.GetAtoms()] + [atom_features(None) for index in pio]
-            f_atoms_prod = [atom_features(mol_prod.GetAtomWithIdx(ri2pi[atom.GetIdx()])) if atom.GetIdx() not in rio else
-                            atom_features(None) for atom in mol_reac.GetAtoms()] + [atom_features(mol_prod.GetAtomWithIdx(index)) for index in pio]
-            
-            if self.reaction_mode in ['reac_diff','prod_diff']:
+            if self.reaction_mode in ['reac_diff','prod_diff', 'reac_prod']:
+                f_atoms_reac = [atom_features(atom) for atom in mol_reac.GetAtoms()] + [atom_features_zeros(mol_prod.GetAtomWithIdx(index)) for index in pio]
+                f_atoms_prod = [atom_features(mol_prod.GetAtomWithIdx(ri2pi[atom.GetIdx()])) if atom.GetIdx() not in rio else
```

Comment:
This is a tricky list construction to understand. Probably could use an explanatory annotation.

Reply:
Can do!

Reply:
I think this is my only note then. When you push that I'll pass the review.

Reply:
Just pushed!